src/dwfl_module.cc

```diff
@@ -10,6 +10,15 @@ extern "C" {
 }
 
 namespace ddprof {
+
+static const char *get_filename(const char *full_path) {
```

Comment:
Great!  This won't work for relative paths, but such paths are never returned by the interfaces we use (those interfaces have no concept of "current working directory" since they're both handled by the kernel), so this is good.